### PR TITLE
Translate forms to English

### PR DESCRIPTION
**Pull request by eric-brechemier**
I have changed the language to "fr" in layout.jade and added an English translation in a separate folder, 'en/'.

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -14,10 +14,18 @@ app.get('/', function(req, res) {
     res.render('form');
 });
 
+app.get('/en', function(req, res) {
+    res.render('form');
```
Comment (risseraka) [inline lib/app.js]: shouldn't it be `res.render('en/form')`?

File: lib/app.js [hunk 2]
```diff
@@ -14,10 +14,18 @@ app.get('/', function(req, res) {
...
+});
+
 app.get('/sponsoring', function(req, res) {
     res.render('sponsoring');
 });
 
+app.get('/en/sponsoring', function(req, res) {
+    res.render('sponsoring');
```
Comment (risseraka) [inline lib/app.js]: same as above, `res.render('en/sponsoring')`?

File: lib/app.js (hunk 1 above)
Comment (eric-brechemier) [inline lib/app.js]: probably, yes.

File: lib/app.js (hunk 2 above)
Comment (eric-brechemier) [inline lib/app.js]: I'll make the change.

**Comment by bobylito** [comment]
LGTM :) Why no merge, @risseraka ?

**Comment by risseraka** [comment]
I don't have permissions, easy as that.

**Comment by bobylito** [comment]
Let me help you with that then :)

**Comment by risseraka** [comment]
You're a good sport, thanks.

**Comment by risseraka** [comment]
Tested, it works fine.

How do we plan to use this on parisjs.org or wiki?
Routes are:
- `/` : talk submition form, French;
- `/sponsoring` : sponsoring submition form, French;
- `/en/` : talk submition form, English;
- `/en/sponsoring` : sponsoring submition form, English.

**Comment by risseraka** [comment]
I think I'll merge it first as it doesn't break anything.
Will swiftly add a localised link to our wiki and see if we've got other ideas to support it on parisjs.org (links to switch or something) later.

**Pull request merged**